installer/ApplicationSetupUtility.php

```diff
@@ -337,13 +339,18 @@ public static function writeSymfonyDbConfigFile() {
 		$dbOHRMUser = $_SESSION['dbInfo']['dbUserName'];
 		$dbOHRMPassword = $_SESSION['dbInfo']['dbPassword'];
 	}
-    
-    $dsn = "mysql:host=$dbHost;dbname=$dbName";
-    $testDsn = "mysql:host=$dbHost;dbname=test_$dbName";
-    
-    if (is_numeric($dbHostPort)) {
-        $dsn = "mysql:host=$dbHost;port=$dbHostPort;dbname=$dbName";
-        $testDsn = "mysql:host=$dbHost;port=$dbHostPort;dbname=test_$dbName";
+
+	if ($_SESSION['dbInfo']['dbHostPortModifier'] == 'socket') {
+        $dsn = "mysql:unix_socket=$dbHostPort;dbname=$dbName";
+        $testDsn = "mysql:unix_socket=$dbHostPort;dbname=test_$dbName";
```

Comment:
What happen OS not support unix_socket ?

Reply:
Can't continue the installation after the second step of web installer.